lsp/src/uri0.ml

```diff
@@ -29,26 +29,22 @@ let of_path path =
   let path = if !Private.win32 then backslash_to_slash path else path in
   Uri_lexer.of_path path
 
+let is_letter c = (c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z')
```

Comment:
just fyi, this could be `let is_letter = function 'a'..'z' | 'A'..'Z' -> true | _ -> false`

Reply:
Oh yes, I like it more. Thanks 🤗